### PR TITLE
bootstrap standard document

### PR DESCRIPTION
**Pull request by tomkralidis**
Bootstrap of profile document.  A PDF render is [available here](https://github.com/user-attachments/files/17013311/rodeo-edr-profile.pdf) for reviewing in context.

### CODE REVIEW

**Comment by havardf** [comment]
Hm, we are probably going to have multiple profiles. As of now, I have a folder like `profiles/observations`. The standard documents is in a tree structure from the `/standard` folder. Do you have any thoughts on how to structure this? Multiple profiles in one document? Or multiple document structures?

**Comment by tomkralidis** [comment]
I would suggest a single profile document with 1..n Requirements Classes (and their associated Conformance Classes).  I am guessing between observations, radar, etc., there will be some commonality between them that can act as the "Core" Requirements baseline.  A single profile document would also reduce document/boilerplate overhead.

If we find that a single profile document becomes too large then we can break out accordingly.

File: .github/workflows/specification.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - name: update gh-pages branch and publish
+        run: | 
+          git checkout gh-pages
+          git config --global user.email "tomkralidis@gmail.com"
+          git config --global user.name "Tom Kralidis"
+          rm -rf standard
+          mv -f /tmp/rodeo-edr-profile/* .
+          git add .
+          git commit -am "update rodeo-edr-profile build"
+          git push
```
Comment (havardf) [inline .github/workflows/specification.yml:48]: Not sure what is going on here:
- Do you setup a branch called gh-pages manually, and the push to that?
- Why have separate step to checkout gh-pages and then checkout again in the `update gh-pages branch and publish` step?
- You set the git config with your email here. Is this just some boilerplate thing? Will these settings be used for anything? Like add name and e-mail in the finished rendered standard doc?

**Comment by havardf** [comment]
> I would suggest a single profile document with 1..n Requirements Classes (and their associated Conformance Classes). I am guessing between observations, radar, etc., there will be some commonality between them that can act as the "Core" Requirements baseline. A single profile document would also reduce document/boilerplate overhead.
> 
> If we find that a single profile document becomes too large then we can break out accordingly.

Sounds good. Not exactly sure what will be common across all profiles at the moment, but seems like a good initial plan.

File: .github/workflows/specification.yml (hunk 1 above)
Comment (tomkralidis) [reply .github/workflows/specification.yml]: - The branch `gh-pages` needs to exist prior to this PR being merged
- double checkout fixed
- git config updated

Comment (tomkralidis) [reply .github/workflows/specification.yml]: @havardf FYI I've setup the `gh-pages` branch.  At this point the repo settings need to enable GitHub Pages, pointing to the `gh-pages` branch.

File: .github/workflows/specification.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'standard/**'
+
+env:
+  SPEC_FILE_BASENAME: rodeo-edr-profile-DRAFT
```
Comment (havardf) [inline .github/workflows/specification.yml:11]: What is the difference between the SPEC_FILE and KPI_FILE?

Comment (tomkralidis) [reply .github/workflows/specification.yml:11]: copy/paste fumble. Fixed.

**Pull request merged**